FitGoal/FitGoal/ImageFetcher/ImageFetcher.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  ImageFetcher.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 4/6/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+var imageCache: [URL: UIImage] = [:]
+
+protocol ImageFetcherDelegate: AnyObject {
+    
+    func imageFetcherIsInProgress()
```

Comment:
Improve the name of this function to match the other functions in the protocol. It should indicate that the fetching operation started.